w-books/src/App/components/Book/styles.css

```diff
@@ -0,0 +1,27 @@
+.Book {
+  display: flex;
+  flex-direction: column;
+  align-content: flex-start;
+  margin: 26px 35px;
+  width: 128px
+}
+
+.Book-title {
+  margin: 11px 0 0;
```

Comment:
other suggestion, if just need change a simple margin side, in this case `top`, you can use `margin-top: 11px`